### PR TITLE
Fix ExGetCurrentProcessorCounts

### PR DESCRIPTION
**Pull request by EggsyCRO**
Fix values written to ThreadKernelTime and TotalCpuTime being swapped.

### CODE REVIEW

**Review by learn-more — DISMISSED** [review]
Hello,
Thanks for your interest in ReactOS!
We ask our contributors [to use their full name when committing](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations).
[Please amend your commit with your full name](https://stackoverflow.com/a/3042512), and update this PR.

To change this for future PR's, you can update it for globally (for the system):
```
git config --global user.name "Your Name"
git config --global user.email you@example.com
```

Or just for the current repository:
```
git config user.name "Your Name"
git config user.email you@example.com
```

Also it's recommended to set up your GitHub account correctly, so maintainers can use Squash and Merge button without getting the author field modified back to nickname by GitHub, it's done in two steps:

1. Set your full name in [GitHub profile settings](https://github.com/settings/profile)
1. Uncheck `Keep my email addresses private` in [GitHub email settings](https://github.com/settings/emails)

Regards,

**Review by HBelusca — COMMENTED** [review]
LGTM otherwise.

**Review by tkreuzer — APPROVED** [review]

**Review by encodedpr — APPROVED** [review]
good

**Review by HBelusca — APPROVED** [review]
OK except for patch author name.

**Review by JoachimHenze — APPROVED** [review]

File: ntoskrnl/ex/sysinfo.c [hunk 1]
```diff
@@ -350,8 +350,8 @@ ExGetCurrentProcessorCounts(PULONG ThreadKernelTime,
 
     Prcb = KeGetCurrentPrcb();
 
-    *ThreadKernelTime = Prcb->KernelTime + Prcb->UserTime;
-    *TotalCpuTime = Prcb->CurrentThread->KernelTime;
+    *ThreadKernelTime = Prcb->CurrentThread->KernelTime;
+    *TotalCpuTime = Prcb->KernelTime + Prcb->UserTime;
     *ProcessorNumber = KeGetCurrentProcessorNumber();
```
[the remark this answers is not in the document]
Comment (ThFabba) [reply ntoskrnl/ex/sysinfo.c:355]: The cast shouldn't be needed, but agreed -- the current thread can move to a different processor at any time during this code, so we should return the number of the processor described by `Prcb` to be consistent with the other data.

**Review by HBelusca — APPROVED** [review]
Approved, but this needs to be done:

> Also it's recommended to set up your GitHub account correctly, so maintainers can use Squash and Merge button without getting the author field modified back to nickname by GitHub, it's done in two steps

**Pull request merged**